dataService.js

```diff
@@ -0,0 +1,37 @@
+//import required json arrays
+const allCategories = require('./src/data/categories.json').categories;
+const allProducts = require('./src/data/products.json').products;
+
+//map individual product objects to product id attributes
+function getProducts(product){
+	productMap[product.id] = product;
+}
+
+//map categoryName as a value for attribute for categoryId
+function getCategories(category){
```

Comment:
Is this line formatted? Needs an space between ) and {. 
You can do it using ESlint automated script. Check and format all the lines